### ISSUE TITLE
Design of Approval page

### ISSUE THREAD

**Issue by jrubenoff** [dobtco/dispatch#29]

**Comment by ajb** [comment]
Maybe I can shed a little more light on this page. 

From Slack:

> Pending opps are ones that have been approved, but haven't been posted because their `publish_date` is in the future.

## 

I originally wanted the "Approve" page to only be visible to approvers, but when I did that, staff members were not able to find their opportunities that were not yet posted. (Since we don't have a "my opps" page.) So I changed the "Approve" page to a generic "Pending" page, and conditionally changed the name of the page based on the user's permissions.

Maybe the best solution is to display a "My opportunities" section on the User profile page?

**Comment by ajb** [comment]
https://dobt.slack.com/archives/client-philly/p1461805261000049

tl;dr

"My opportunities" on account page
"Approve" page is for approvers only

**Comment by ajb** [comment]
This got mixed up with some other stuff, so I ended up pulling it into `master`.

Here's what it looks like:

**I explained the purpose of the page by using copywriting, both in the blank slate and non-blank slate:**

![img](http://take.ms/NNmdR)

![img](http://take.ms/4qvMH)

**The "my opportunities" section is sorted by recently updated, and includes the status of each opp:**

![img](http://take.ms/zlFAu)
